web/conf/rhn_web.conf

```
@@ -110,6 +110,12 @@ ssh_push_sudo_user =
 # This is potentially dangerous and should not be used
 ssh_push_task_timeout = 0
 
+# preflight script to run on ssh push minions to deploy Salt Bundle
+ssh_salt_pre_flight_script = /usr/share/susemanager/salt-ssh/preflight.sh
+
+# use salt-thin as a fallback method if Salt Bundle is not available
+ssh_use_salt_thin = false
```

As we just discussed, we would need to set this as `true` for now right? As we won't have the necessary code in the Salt version shipped in SLE15SP4 public beta.

```suggestion
ssh_use_salt_thin = true
```

---

Well, in fact this is not yet clear. We still need to clarify what we want to do